Source/Schema.NET.Tool/ViewModels/Class.cs

```diff
@@ -150,43 +150,72 @@ public override string ToString()
 
             stringBuilder.AppendIndentLine(4, "{");
 
-            if (string.Equals(this.Name, "Thing", StringComparison.OrdinalIgnoreCase))

```

Comment:
Any reason for removing this property?

Reply:
If it exists, it leads to an unbuildable project. `Context` already exists on `JsonLdObject` and is type `JsonLdContext`. The override declares the type as `string` (this is why it fails) and the schema URL. On top of that, `JsonLdContext` already defines the default value for the context with the HTTPS URL.

So even sorting the type issue, it seems a little redundant having `Thing` specifically set a value that is already defined by default.